### ISSUE TITLE
Add indicator links to the overview page.

### ISSUE THREAD

**Issue by tildechris** [cmu-delphi/www-covidcast#828]
Currently we say: 

```
Our indicators include our national COVID survey, public experience, attitude, behavior, doctor's visits, hospital admissions, tests, cases and deaths.

COVID-19 INDICATORS
[Explore our API]

* 20 Survey Indicators
* 6 Public’s Behavior Indicators
* 3 Early Indicators
* 4 Late Indicators
```

We should deep link into different indicators from this page link we do for locations.  We can also define what indicators are in the first sentence.  Finally, we should link to sections in the survey results page.

COVID-19 INDICATORS

COVIDcast gathers data from dozens of sources and produces a set of indicators which can inform our reasoning about the pandemic. Indicators are produced from these raw data by extracting a metric of interest, tracking revisions, and applying
additional processing like reducing noise, adjusting temporal focus, or enabling more direct comparisons.

Our most useful indicators are visualized in this site, but for the full set, please [visit our API](https://cmu-delphi.github.io/delphi-epidata/).

* 6 **Public’s Behavior Indicators** like [People Wearing Masks](), [Vaccine Acceptance]() and [At Away Location 6hr+](),
* 3 **Early Indicators** like [COVID-Like Symptoms in Community]() and [Doctor's Visits]()
* 4 **Late Indicators** like [Cases](), [Deaths](), [Hospital Admissions](), [Tests]()

DELPHI PANDEMIC SURVEY VIA FACEBOOK

The U.S. Pandemic Survey offers insights into public sentiment on:

* [COVID-Like Symptoms]()
* [Mask Use]()
* [Vaccines]()
* [Social Distancing]()
* [Mental Health]()
* [Testing]()

|53,000|18,324,000|
|---|---|
|Daily survey participants|Total survey responses|

**Comment by tildechris** [comment]
> I will get approval/consensus on the indicator definition wording.

@krivard mentioned that she will have time to get us this definition this afternoon.

**Comment by sgratzl** [comment]
> 
> * 20 Survey Indicators: Mask Use, Vaccine Attitudes, Social Distancing, and Mental Health
> * 6 Public’s Behavior Indicators: Away from Home Mobility,
> * 3 Early Indicators: COVID-like Symptoms and Doctor's Visits
> * 4 Late Indicators: Cases, Deaths, Hospital Admissions, Tests

which signal is `Social Distancing`? we have a bunch of them. `Mental Health` we currently don't have a regular indicator about this, just a few in the survey results

**Comment by tildechris** [comment]
Yeah, I was conflating surveys and indicators.

```
20 Survey Indicators: Mask Use, Vaccine Attitudes, Visits to bars & restaurants
6 Public’s Behavior Indicators: Away from Home Mobility
3 Early Indicators: COVID-like Symptoms and Doctor's Visits
4 Late Indicators: Cases, Deaths, Hospital Admissions, Tests
```

Perhaps we should also add the same treatment to the surveys section and link to the start of each section:

```
Explore responses on Symptoms, Mask Use, Vaccines, Social Distancing, Mental Health & Testing or
Go to the survey dashboard.
```

**Comment by krivard** [comment]
It looks like we're using `indicator` here in the sense of "one or more signals describing a single metric computed from a single data source" -- the cases indicator includes smoothed/raw, incidence/cumulative, and counts/ratios variations, but they all describe the volume of cases, and they are all distinct from the deaths indicator even though they are generated using data from the same source organization.

Truncate the following to suit the desired level of detail:

```
Delphi gathers data from dozens of sources and produces a set of indicators
which can inform our reasoning about the pandemic. Each indicator measures one
thing, but may have multiple options for reducing noise, adjusting temporal
focus, or enabling more direct comparisons. Our most useful indicators are
visualized in this site, but for the full set, please visit our API.
```

**Comment by tildechris** [comment]
@sgratzl I updated the description, PTAL.

**Comment by sgratzl** [comment]
fixed with #833
